ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/http3/Http3FrameAdapter.kt

```
@@ -0,0 +1,31 @@
+        while (true) {
+            val message = receive()
+            val content = message.content() ?: Unpooled.EMPTY_BUFFER
+
+            try {
+                transferByteBuf(content, bc)
+            } finally {
+                content.release()
+            }
+        }
+    } catch (closed: ClosedReceiveChannelException) {
+    } catch (t: Throwable) {
```

detekt (1.23.8)

[warning] Empty catch block detected. If the exception can be safely ignored, name the exception according to one of the exemptions as per the configuration of this rule.

(detekt.empty-blocks.EmptyCatchBlock)